contrib/99designs/gqlgen/option.go

```diff
@@ -0,0 +1,49 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package gqlgen
+
+import (
+	"math"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
+)
+
+const defaultServiceName = "gqlgen"
```

Comment:
Can we make the service name `graphql`. This would better align with the python, node, and .net implementations
```suggestion
const defaultServiceName = "graphql"
```

Reply:
Yes thanks for catching that.